app/models/abuse_report.rb

```diff
@@ -1,86 +1,76 @@
+  validates_presence_of :language
+  validates_presence_of :summary
   validates_presence_of :comment
   validates_presence_of :url
-  validates :email, :email_veracity => {:allow_blank => true}
-  attr_accessor :cc_me
-  validates :email, :presence => {:message => ts("cannot be blank if requesting an emailed copy of the Abuse Report")}, :if => "email_copy?"
   validate :work_is_not_over_reported
+  validates_length_of :summary, maximum: ArchiveConfig.FEEDBACK_SUMMARY_MAX,
+                                too_long: ts('must be less than %{max}
+                                             characters long.',
+                                max:
+                                ArchiveConfig.FEEDBACK_SUMMARY_MAX_DISPLAYED)
 
-  scope :by_date, order("created_at DESC")
+  scope :by_date, order('created_at DESC')
 
   attr_protected :comment_sanitizer_version
 
   # if the URL ends like "works/123", add a / at the end
   # if the URL contains "works/123?", remove the parameters and add a /
-  # work_is_not_over_reported uses the / so "/works/1234" isn't a match for "/works/123"
+  # work_is_not_over_reported uses the / so "/works/1234" isn't a match
+  # for "/works/123"
   before_validation :clean_work_url, on: :create
   def clean_work_url
     if url.match(/(works\/\d+)$/)
-      self.url = url + "/"
+      self.url = url + '/'
     elsif url.match(/(works\/\d+\?)/)
-      self.url = url.split("?").first + "/"
+      self.url = url.split('?').first + '/'
     else
       url
     end
   end
 
-  def email_copy?
-   cc_me == "1"
-  end
-
-  app_url_regex = Regexp.new('^https?:\/\/(www\.)?' + ArchiveConfig.APP_HOST, true)
-  validates_format_of :url, :with => app_url_regex, :message => ts('does not appear to be on this site.')
-
-  # Category names for form
-  CATEGORIES = [
-    ["Children's Online Privacy and Protection Act", 11468],
-    ["Reproduction of copyrighted or trademarked material (unfair use)", 11469],
-    ["Illegal or non-fanwork content", 11471],
-    ["Plagiarism", 11470],
-    ["Open Doors", 11516],
-    ["Harassment", 11473],
-    ["Next-of-kin claim", 11514],
-    ["Personal information (outing)", 11472],
-    ["Spam or commercial promotion", 11515],
-    ["Inappropriate content rating", 11475],
-    ["Insufficient content warning", 11476]
-  ]
+  app_url_regex = Regexp.new('^https?:\/\/(www\.)?' +
+                             ArchiveConfig.APP_HOST, true)
+  validates_format_of :url, with: app_url_regex, message: ts('does not appear to
+                                                              be on this site.')
 
   def email_and_send
     AdminMailer.abuse_report(id).deliver
-    if email_copy?
-      UserMailer.abuse_report(id).deliver
-    end
+    UserMailer.abuse_report(id).deliver
     send_report
   end
 
   def send_report
-    return unless %w(staging production).include?(Rails.env)
+    return unless %w(staging production development).include?(Rails.env)
```

Comment:
The development environment snuck back in in your last commit. /o\ I think this pull request might be slightly cursed.